prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -142,11 +135,11 @@ def _verify_redfish_call(self, uri: str) -> Optional[Dict[str, Any]]:
 
         Returns None if URI isn't present.
         """
-        resp: RestResponse = self.redfish_obj.get(uri)
-        if resp.status == 200:
-            return resp.dict
-        logger.debug("Not able to query from URI: %s.", uri)
-        return None
+        resp = self.redfish_obj.get(uri)
+        if resp.status != 200:
+            logger.debug("Not able to query from URI: %s.", uri)
+            return None
+        return resp.dict
```

Comment:
I think there is no big different now. In other situation, we may do more thing when getting successful return code. It's easier to extend the code in the future. I guess.
ex:
```
    if resp.status != 200:
        return None
        return resp.dict
    # continue to do something.... 
    return resp

Reply:
Yeah, as @Pjack mentioned, it has no difference now, but in other cases it will be beneficial. In the end, I would consider those as "bait out" so we don't do anything after that.

Reply:
@dashmage maybe you can check: https://www.youtube.com/watch?v=rrBJVMyD-Gs

Reply:
I see what you mean, thanks for the info. I'll resolve this conversation :)